ydb/core/util/actorsys_test/single_thread_ic_mock.cpp

```diff
@@ -1,6 +1,6 @@
 #include "single_thread_ic_mock.h"
 #include "testactorsys.h"
-#include "stlog.h"
+#include "../stlog.h"
```

Comment:
Maybe it's better to write full path if the include is not in the same directory?